templates/swagger/v1_json.tmpl

```diff
@@ -21036,4 +21149,4 @@
       "TOTPHeader": []
     }
   ]
-}
+}
```

Comment:
A long-standing question in my mind (just curious), sometimes the last EOL appears, sometimes it disappears ... 😂

Reply:
I'm guessing some differences in OSes when running generate swagger func 😢

Reply:
Could add a [small sed](https://unix.stackexchange.com/a/31955/221810) at the end of the generation process.

Reply:
Interestingly that exact sed is [in the codebase](https://github.com/go-gitea/gitea/blob/8cc60ee01a6af4e1691fc49252ce2c79efd320dc/Makefile#L321) since 4 years.

Reply:
Can't see any issue with the newline addition in the makefile. A file without newline should not be able to pass `swagger-check`.

Reply:
Issue appeared again in [other generated file](https://github.com/go-gitea/gitea/pull/24406#issuecomment-1527638257). I'm kind of out of ideas how it happens.